eval/vals/repr.go

```diff
@@ -32,6 +32,8 @@ type Reprer interface {
 // format "<unknown %v>".
 func Repr(v interface{}, indent int) string {
 	switch v := v.(type) {
+	case Nil:
```

Comment:
This is not a builtin type and thus doesn't need special-casing. Implement `Reprer` for `Nil` (or rather `JsonNil`, the name I suggested).

Reply:
Will do.